### DISCUSSION TITLE
DatePicker Field Basic/Pro Predefined Dates?

### DISCUSSION THREAD

**Discussion by madsem** [formkit/formkit#1460] [General]
Hey there gang,

is it possible to extend the DatePicker input to display predefined dates? Like today, yesterday, MTD and so forth?

**Comment by justin-schroeder** [comment]
Are you thinking like a little panel with some you could select from?

**Reply by madsem** [reply]
Exactly 👍 

Could be anywhere, inside a dropdown, or on either side / footer. Upon clicking, it would then select the predefined range and close the picker.

**Reply by justin-schroeder** [reply]
I think its a solid idea!
